src/pages/aspect/AspectSettingsPage.tsx

```diff
@@ -33,6 +34,9 @@ const AspectSettingsPage: FC<AspectSettingsPageProps> = ({ paths: _paths }) => {
       references: aspect?.references,
     };
 
+  const aspectIndex = resolved.pathname.indexOf('/aspects');
+  const contributeUrl = resolved.pathname.substring(0, aspectIndex);
```

Comment:
Did you try `navigate('..')`?

Reply:
`..` will navigate you just one step back - we want three steps instead.

Reply:
`"../../.."`

Reply:
But if we just want to go to a specific route rather than "go up" (not sure if it's the case), I'd rather build a route from scratch.

Reply:
it happened to me several times, but now it stopped. I am removing it and we will see.